src/ui/flutter_app/lib/l10n/resources.dart

```diff
@@ -19,11 +19,11 @@
 import 'dart:io';
 
 import 'package:flutter/material.dart';
-import 'package:sanmill/common/config.dart';
-import 'package:sanmill/common/constants.dart';
+import 'package:sanmill/shared/common/config.dart';
+import 'package:sanmill/shared/common/constants.dart';
 import 'package:sanmill/generated/l10n.dart';
-import 'package:sanmill/style/app_theme.dart';
-import 'package:sanmill/widgets/list_item_divider.dart';
+import 'package:sanmill/shared/theme/app_theme.dart';
+import 'package:sanmill/screens/list_item_divider.dart';
```

Comment:
Why is list_item_divider a type of screen? Or is it to indicate that it is an element in the screen?